frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -19,19 +20,23 @@ export type RevisionsOverviewListItemProps = {
 };
 
 const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
-  revision: {
+  revision,
+  service,
+}) => {
+  const {
     metadata: { name, namespace },
-  },
-  service: {
+  } = revision;
+  const {
     status: { traffic },
-  },
-}) => {
-  const getTraffic = (revision: string) => {
+  } = service;
+  const getTraffic = (name: string) => {
     if (!traffic || !traffic.length) {
       return null;
     }
-    return `${_.get(_.find(traffic, { revisionName: revision }), 'percent', 0)}%`;
+    return `${_.get(_.find(traffic, { revisionName: name }), 'percent', 0)}%`;
```

Comment:
The default value here indicates that `0%` traffic is assigned to a revision whereas in this case, it's not assigned any percentage if it's not in `status.traffic`. Maybe return `null` as default value here.